src/config.ts

```diff
@@ -80,11 +80,7 @@ const validators = {
   REACT_APP_FEATURE_BNBSMARTCHAIN: bool({ default: false }),
   REACT_APP_FEATURE_POLYGON: bool({ default: false }),
   REACT_APP_FEATURE_GNOSIS: bool({ default: false }),
-  REACT_APP_FEATURE_ZRX_AVALANCHE: bool({ default: false }),
-  REACT_APP_FEATURE_ZRX_BNBSMARTCHAIN: bool({ default: false }),
-  REACT_APP_FEATURE_ZRX_POLYGON: bool({ default: false }),
-  REACT_APP_FEATURE_ZRX_ETHEREUM: bool({ default: false }),
-  REACT_APP_FEATURE_ZRX_OPTIMISM: bool({ default: false }),
+  REACT_APP_FEATURE_ZRX_SWAP: bool({ default: false }),
```

Comment:
Writing this comment as I'm starting to review this so it may become obvious as I continue reading through the implementation, but how are we going to do flags to support a specific network for a specific swapper?